recipe/meta.yaml

```diff
@@ -1,51 +1,66 @@
-{% set version = "5.9.8" %}
+{% set version = "6.6.1" %}
 {% set ver = '.'.join(version.split('.')[:2]) %}
 
 package:
-  name: qtserialport
+  name: qt6-serialport
```

Comment:
@jschueller this would harmonize the name of the package with that of other qt6 packages. however, should I just make a new feedstock for this and archive this one?

Reply:
yes, that would be great

Reply:
ok understood. my understanding is that you would prefer:

* close this feedstock + creating a new one through staged-recipes